draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -688,6 +688,13 @@ If the IP range that the server expects clients to connect from is restricted, t
 If every RADIUS/(D)TLS client is configured with an IP range, then the server does not even have to perform a partial TLS handshake if the connection attempt comes from outside every allowed range, but can instead immediately drop the connection.
 To perform this lookup efficiently, RADIUS/(D)TLS servers SHOULD keep a list of the accumulated permitted IP address ranges, individually for each transport.
 
+## TLS Session Lifetime and Key Rotation
+
+The underlying TLS sessions of RADIUS/(D)TLS connections may have a long lifetime.
+Especially when dealing with high volume of RADIUS traffic, the encryption keys have to be rotated regularly, as described in {{RFC8446, Section 5.5}}.
```

Comment:
Perhaps explain when they're rotated?

```
... the encryption keys have to be rotated regularly, depending on both the amount of data which was transferred, and on the encryption method.  See {{RFC8446, Section 5.5}} for more information.
```

Reply:
Sounds good. I've also added the reference to the CFRG draft on AEAD limits